### ISSUE TITLE
Could not find generator

### ISSUE THREAD

**Issue by noomerikal** [doorkeeper-gem/doorkeeper#676]
Can't find generator, am I missing something? Tried both 2.2.1 and 3.0.0.rc1.

```
± % rails g doorkeeper:install
Could not find generator 'doorkeeper:install'. Maybe you meant 'rspec:install' or 'devise:install' or 'simple_form:install'
```
Rails 4.2.1
ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14]

**Comment by noomerikal** [comment]
Got it resolved - I updated RubyGems from 2.4.5 -> 2.4.8, bundle update, and then stopped spring. I think it was stopping spring that worked since the bundle update was mostly minor releases.
```
gem update --system
bundle update
spring stop
```

**Comment by 3pns** [comment]
Same problem with latest rails 6.0.0.rc1, it seems like it is required to stop spring to run the install command, it is neither specified in the main github page/rails install guide/doorkeeper-provider-app example project.

~~~ console
$ rails generate doorkeeper:install
Running via Spring preloader in process 54002
Could not find generator 'doorkeeper:install'. Maybe you meant "erd:install"?
Run `rails generate --help` for more options.
$ spring stop
Spring stopped.
$ rails generate doorkeeper:install
Running via Spring preloader in process 54049
      create  config/initializers/doorkeeper.rb
      create  config/locales/doorkeeper.en.yml
       route  use_doorkeeper
===============================================================================

There is a setup that you need to do before you can use doorkeeper.

Step 1.
Go to config/initializers/doorkeeper.rb and configure
resource_owner_authenticator block.

Step 2.
Choose the ORM:

If you want to use ActiveRecord run:

  rails generate doorkeeper:migration

And run

  rake db:migrate

Step 3.
That's it, that's all. Enjoy!

===============================================================================
~~~
